README.md

```diff
@@ -79,13 +81,20 @@ To run the app locally, first clone this repository and `cd` into its directory.
     python manage.py runserver
```

Comment:
we should consider moving this to python3

Reply:
@rojasTob I'm now getting `ModuleNotFoundError: No module named 'bootstrap4'`. Is this working for you?

Reply:
Hey @bharatmurali. Yes, this is working in my local. However, I've modified the step `6.` in the Readme file, it seems like the virtual env was not activated.